SPECS/azurelinux-sysinfo/azurelinux-sysinfo.spec

```diff
@@ -13,6 +13,7 @@ Source2:        azurelinux-sysinfo.service
 Source3:        sysinfo-selinuxpolicies.cil
```

Comment:
Bump `Release` number?

Reply:
Good catch!

On this topic... @dmcilvaney @PawelWMS have we ever had a github action that checks for an inadvertently forgotten release bump? (i.e., Should be able to rpmspec-parse the .spec in the _from_ branch and the _to_ branch and at least require a difference.) Missing a bump can be problematic, and we should definitely make machines sort this out for us. 

(And before it's mentioned, I assume there's a problem of cases where someone is just updating changelog entries or making changes to comments, say, and that shouldn't strictly require a bump. That sounds like a case where someone could use an escape hatch of adding some well-known tag/line in the PR description that quiets the check. e.g., `IKnowWhatIAmDoingAndReallySwearUpAndDownThatThereAreNoMaterialChangesInThisSpecFile`. Or maybe something shorter if someone clever can come up with something 😄  )

Reply:
Thank you for catching this!